scss/_variables.scss

```diff
@@ -1514,9 +1514,7 @@ $accordion-body-padding-x:                $spacer !default; // Boosted mod
 
 $accordion-button-padding-y:              $accordion-padding-y !default;
 $accordion-button-padding-x:              $accordion-padding-x !default;
-// fusv-disable
```

Comment:
Normally it shouldn't be too much of a problem for our users to re-enable this var